backend/inspirehep/mailing/providers/mailtrain.py

```diff
@@ -40,3 +42,29 @@ def mailtrain_subscribe_user_to_list(list_id, email, first_name, last_name):
             "User successfully added to the mailing list.", list_id=list_id, email=email
         )
     return response
+
+
+def mailtrain_update_weekly_campaign_content(html_content):
+    """Sends html_content of the campaign to redis WEEKLY_JOBS_EMAIL_REDIS_KEY.
+    Args:
+        html_content (str): HTML content of the campaign.
+    """
+    mailtrain_key = current_app.config.get("WEEKLY_JOBS_EMAIL_REDIS_KEY")
+    if not mailtrain_key:
+        LOGGER.error(
+            "`WEEKLY_JOBS_EMAIL_REDIS_KEY` is not set. MAILTRAIN campaign is not properly configured!"
+        )
+        return
+
+    redis_url = current_app.config.get("CACHE_REDIS_URL")
+    redis = StrictRedis.from_url(redis_url, decode_responses=True)
+    date = datetime.datetime.now().timestamp()
+    title = current_app.config.get(
+        "WEEKLY_JOBS_EMAIL_TITLE", "Weekly update on new job listings."
+    )
+    data = {"date": date, "title": title, "html": html_content}
+    result = redis.hmset(mailtrain_key, data)
+    if not result:
+        LOGGER.error("Redis was not able to store html_content of weekly campaign!")
```

Comment:
Redis offers response. True if everything is ok, False if not. Actually in docs there is no explanation when it can return False, propably only when redis server will return False as the response is direct response form Redis server:
`return self.execute_command('HMSET', name, *items)` 
https://redis-py.readthedocs.io/en/latest/_modules/redis/client.html#Redis.hmset